crates/cheatcodes/src/inspector.rs

```diff
@@ -83,14 +84,6 @@ pub struct BroadcastableTransaction {
 /// List of transactions that can be broadcasted.
 pub type BroadcastableTransactions = VecDeque<BroadcastableTransaction>;
 
-#[derive(Clone, Debug)]
-pub struct AccountAccess {
-    /// The account access.
-    pub access: crate::Vm::AccountAccess,
-    /// The call depth the account was accessed.
-    pub depth: u64,
-}
```

Comment:
I've removed this wrapper type. It's not necessary since `depth` is now a field on the underlying `AccountAccess` type. I'm happy to reverse this change if there are plans to use this wrapper for some other purpose, but as of now, it seems redundant to me.